docs/security/host-certs.md

```diff
@@ -60,20 +60,62 @@ If your institution is not in the list and Let's Encrypt certificates do not mee
 
 1. Generate a Certificate Signing Request (CSR) and private key:
 
+1. Create a custom openssl config for this, using your institution's information for the different fields in the
+   `[ dn ]` section, and the desired hostname for the `CN` field.
+
+        :::console
+        # mycsr.conf
+        [ req ]
+        default_bits = 2048
+        prompt = no
+        encrypt_key = no
+        default_md = sha256
+        distinguished_name = dn
+
+        [ dn ]
+        DC = org
+        DC = incommon
+        C = US
+        postalCode = 53706
+        ST = WI
+        L = Madison
+        street = 1210 West Dayton Street
+        O = University of Wisconsin-Madison
+        OU = OCIS
+        CN = voms.opensciencegrid.org
+
+
+1. If you need to generate a CSR with Subject Alternative Names (SAN), add these lines into your config:
```

Comment:
This needs some exposition

1. Let's link to the wikipedia article on SANs (or some other authoritative source)
1. Should every SAN have the following line?

        subjectAltName = DNS: voms1.opensciencegrid.org, DNS: abc.opensciencegrid.org, DNS: def.opensciencegrid.org

    Let's use `%RED%DNS: voms.opensciencegrid.org, DNS: <SAN 2>...DNS: <SAN N>%ENDCOLOR%` or something similar and reference it in the expository text

Reply:
> Should every SAN have the following line?

Apparently there are two ways to do it - the above method with all the SANs on a single line was suggested by the [ehitter page](https://ethitter.com/2016/05/generating-a-csr-with-san-at-the-command-line/), but the [cacert wiki page](http://wiki.cacert.org/FAQ/subjectAltName) gives another way to do it with each SAN DNS entry on a separate line.  I actually like the 1-per-line method a little better, but i was going off the ehitter example first.

Note that the hostname in the `CN` does not need to be listed in the SANs, iiuc.

Reply:
My concern isn't the single vs multiline formatting, but rather the clarity of the example SANs: we use highlighted text wrapped in angle brackets for text that the user needs to replace in other docs, let's do that.

And yea, it appears that InCommon automatically prepends the `CN` in the SANs.